shared/candidate-helpers.js

```diff
@@ -30,18 +42,69 @@ module.exports = {
                 newCandidateObj.party = results[i].elections[0].party_affiliation || '';
                 newCandidateObj.incumbent_challenge = results[i].elections[0].incumbent_challenge_full || '';
                 newCandidateObj.election = results[i].elections[0].election_year || '';
-
                 newCandidateObj.state = results[i].elections[0].state || '';
                 newCandidateObj.district = results[i].elections[0].district || '';
+
+                if (typeof results[i].elections[0].primary_committee !== 'undefined') {
+                  newCandidateObj.primary_committee.id = results[i].elections[0].primary_committee.committee_id || '';
+                  newCandidateObj.primary_committee.name = results[i].elections[0].primary_committee.committee_name || '';
+                  newCandidateObj.primary_committee.designation = results[i].elections[0].primary_committee.designation_full || '';
+                  newCandidateObj.primary_committee.url = '/committees/' + results[i].elections[0].primary_committee.committee_id || '';                
+                }
+
+                if (typeof results[i].elections[0].affiliated_committees !== 'undefined') {
+                  var committeesAll = [],
+                      committeesA = [],
```

Comment:
Totally a style thing, but I'd be in favor of switching these three arrays to more descriptive names, even if long. I think it reads a bit clearer that way. `committeesAuthorized` or `authorizedCommittees` and so on in that pattern works, I think.